crates/forge/src/multi_runner.rs

```diff
@@ -391,8 +395,25 @@ impl MultiContractRunnerBuilder {
                 deployable_contracts.insert(id.clone(), (abi.clone(), bytecode, libs_to_deploy));
             }
 
-            if let Some(bytes) = linked_contract.get_deployed_bytecode_bytes() {
-                known_contracts.insert(id.clone(), (abi.clone(), bytes.to_vec()));
+            if !use_zk {
```

Comment:
I think we always want EVM bytecodes, no?

Reply:
Let's discuss it offline